src/erlcloud_ses.erl

```diff
@@ -43,6 +43,13 @@
 -export([verify_email_identity/1, verify_email_identity/2]).
 -export([verify_domain_identity/1, verify_domain_identity/2]).
 
+-export([create_custom_verification_email_template/6, create_custom_verification_email_template/7]).
```

Comment:
Just to clarify on grouping, I was suggesting that `create_custom_verification_email_template` go above `delete_identity`, that `update_custom_verification_email_template` go between `set_identity_notification_topic` and `verify_domain_dkim`, etc. In other words, to me, the cadre of functions you've contributed should be interspersed in the module in function name order.

Reply:
OK. Now I see what you mean. Personally, it's a way I have never like. Alphabetically grouping functions make using an API harder. Grouping by object type (i.e. all the CRUD ops for a specific object together for example) makes it easier to search when looking at the source. That said, I will follow the style of the API as it is currently and intersperse the functions. Now it's very clear to me.

Reply:
I understand the preference, but IMO consistency (especially within a module) is a lot more valuable than just having each contributor organize the code as they see fit. I'd be open to regrouping the module as a whole separately from extending it, but as-is, contributions should be consistent with existing patterns.

Looks like your update of the code blocks is consistent, and just remains to intersperse the exports to be interspersed in symmetric order here at the top.

Last question is, will you make the unit tests, or should we just merge this and raise an issue to do so in the future?